instrumentation/play/play-mvc/play-mvc-2.4/javaagent/build.gradle.kts

```diff
@@ -29,26 +30,56 @@ otelJava {
   maxJavaVersionForTests.set(JavaVersion.VERSION_1_8)
 }
 
+testSets {
+  create("play24Test")
```

Comment:
I think(?) it would be more consistent with others if we can switch and have normal `test` reflect the "base" version 2.4, and have the additional test set for 2.5 (or latest)

Reply:
I think that was what I tried initially. The problem with it is that Intellij doesn't like it when you are pointing latest dep test set to the same directory as an other test set, it will complain about duplicate source root or something like that.

Reply:
thx, I'm good with it as-is 👍